### PR TITLE
Initial django setup on cloud.gov

### PR DESCRIPTION
**Pull request by tbaxter-18f**
With Postgres

### CODE REVIEW

File: nrm_django/Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: python manage.py runserver 0.0.0.0:8080
```
Comment (tbaxter-18f) [inline nrm_django/Procfile:1]: I know this isn't ideal, but it seems adequate for small-group prototyping. I'll log an issue to implement a more robust server.

Comment (neilmb) [reply nrm_django/Procfile:1]: Ticket number here for breadcrumbs when you make it?

**Review by neilmb — APPROVED** [review]
LGTM, my only question is how much of this initial startup is boilerplate and how much is specific to the project. If it was inspired or spun up from a similar setup/tutorial, could we indicate that in the README or the PR somewhere?

File: nrm_django/Procfile (hunk 1 above)
Comment (tbaxter-18f) [reply nrm_django/Procfile:1]: Good idea. #78 is logged for it.

**Comment by tbaxter-18f** [comment]
Let me ponder about doc'ing the setup. This was a combo of the out-of-date example Peter gave, poking 'round docs and looking at some past 18F projects. 

We could maybe do cloud.gov a solid by writing up a blog post or starter project. 

But honestly, it's 95% boilerplate, and 5% figuring out the right bit to get it working.

**Comment by tbaxter-18f** [comment]
Commit resolves #73

**Pull request merged**